GitUpKit/Core/GCDiff.m

```diff
@@ -464,12 +468,11 @@ - (GCDiff*)_diffWithType:(GCDiffType)type
   if (options & kGCDiffOption_IgnoreAllSpaces) {
     diffOptions.flags |= GIT_DIFF_IGNORE_WHITESPACE;
   }
-  if (filePattern) {
+  if (filePattern && filePath) {
     diffOptions.pathspec.count = 1;
-    const char* filePath = GCGitPathFromFileSystemPath(filePattern);
-    diffOptions.pathspec.strings = (char**)&filePath;
+    diffOptions.pathspec.strings = (char **)&filePath;
 
-    static NSCharacterSet* set = nil;
+    static NSCharacterSet *set = nil;
```

Comment:
Apologies, these were done by Clang-Format re: [#1 in CONTRIBUTING](https://github.com/git-up/GitUp/blob/master/CONTRIBUTING.md#L9). See #326, I suppose.